inference-engine/src/mkldnn_plugin/nodes/mkldnn_interpolate_node.cpp

```diff
@@ -1648,273 +1629,341 @@ SizeVector to5Dim(SizeVector casesDim) {
     return dim5;
 }
 
+bool MKLDNNInterpolateNode::isSupportedOperation(const std::shared_ptr<ngraph::Node>& op, std::string& errorMessage) noexcept {
+    try {
+        const auto interp = std::dynamic_pointer_cast<const ngraph::opset4::Interpolate>(op);
+        if (!interp) {
+            errorMessage = "Only opset4 Interpolate operation is supported";
+            return false;
+        }
+        const auto &interpAttr = interp->get_attrs();
+        const auto &interpMode = interpAttr.mode;
+        if (interpMode != ngraph::opset4::Interpolate::InterpolateMode::nearest &&
+            interpMode != ngraph::opset4::Interpolate::InterpolateMode::linear &&
+            interpMode != ngraph::opset4::Interpolate::InterpolateMode::linear_onnx &&
+                interpMode != ngraph::opset4::Interpolate::InterpolateMode::cubic) {
```

Comment:
Please use `one_of` here and in other places, It also would be worthwhile to add  using for `ngraph::opset4::Interpolate::InterpolateMode`

Reply:
done